SpareMe/screens/Home.js

```diff
@@ -86,6 +115,29 @@ export default class Home extends Component {
         this.setState({url: text});
     }
 
+    menuHandler = (value) => {
+        switch(value) {
+          case 1: //Sign In
```

Comment:
You should use constants both here and in the `MenuOption` markup. Something akin to `constants.SIGN_IN`, etc. This makes your code much clearer

Reply:
Done